components/LinkTabs.data.ts

```diff
@@ -0,0 +1,19 @@
+import { LinkTabProps } from './LinkTab'
+
+export const cw20LinkTabs: LinkTabProps[] = [
+  {
+    title: 'Instantiate',
+    description: `Lorem ipsum dolor sit amet consectetur adipisicing elit.`,
+    href: '/contracts/cw20/instantiate',
+  },
+  {
+    title: 'Query',
+    description: `Lorem ipsum dolor sit amet consectetur adipisicing elit.`,
+    href: '/contracts/cw20/query',
+  },
+  {
+    title: 'Execute',
+    description: `Lorem ipsum dolor sit amet consectetur adipisicing elit.`,
```

Comment:
@findolor I'd like to replace this lorem ipsum with real copy.

Could you help me get started? In your own words, can you describe each of these actions (instantiate, query, execute)?

I can help refine and edit.

Reply:
Ok will write some things and share it with you

Reply:
Did some updates in f60e55edad5d6c62b45f1f0b21408fe29799dc10, but do update accordingly.

Reply:
@RyanWarner 

Instantiate: Initialize and create new instances of cw20 contract.
Query: Query any cw20 contract address to get data
Execute: Invoke a method on any cw20 contract address

I think this is a good start, somethings along these lines should do